docs/CHANGELOG.md

```diff
@@ -3,6 +3,11 @@ Version 3.34.0-eisop2 (May ?, 2023)
 
 **User-visible changes:**
 
+The Initialization Checker supports new qualifier:
+* `@PolyInitialized`. By using the @PolyInitialized qualifier,
```

Comment:
Why is this sentence split over three bullet points? It reads rather oddly.
The "acknowledges" is also odd wording. At compile time we don't have objects, so that's also not the word you want to use here. Please try rewording this paragraph.

Reply:
OK. But the manual still uses 'object'(e.g. The Initialization Checker determines whether an object is initialized or not. For any object that is not fully initialized, the Nullness Checker treats its fields as possibly-null — even fields annotated as @NonNull), should we change that as well? @wmdietl

Reply:
The format is changed. I have tried to change the words here. 

I am wondering what are the terminology at compile time and at run time for most java programs. Is there any reference for that?